### ISSUE TITLE
Remove View Members page from View/Edit Roster workflow for instructors

### ISSUE THREAD

**Issue by joliet** [DukeLearningInnovation/kits#571]
I am student trying to create a new kit for my study group but when I try to add other group members to my kit, I don't see how to add new members, just a page with me listed as a member.  This makes me feel confused about how to add my study group members to my kit.

How might we (the Kits project team) make it easier for kit owners to manage members?

**Questions**

**Assumptions**

**Hypotheses**
User testing results showed that the View Mode page is confusing for users.

Acceptance criteria:

1. Instructor/Assistants/Owners/Editors are taken directly to the edit members page when they select View/Edit Members for a course kit.
2. . Students/Viewers are taken directly to the view members page when they select View Members for a course kit.
3. There are no view or edit mode buttons on either the View page or the Edit page.

**Comment by joliet** [comment]
This looks good @noahhh !  I didn't give any acceptance criteria for collaboration kits so that's on me.  But I did test those and noticed that Viewers were unable to see the members.  Viewer should be able to see members in collaboration kits just like students can see members in course kits.  The visitor role should not see members in a kit however, I was having trouble testing that.  I think something else might be going on there.  I added a student I'm working with as a visitor in a few official course kit and they did not show up for her when I impersonated her.  That's a separate problem, though.  I will create a new issue for that.
